### PR TITLE
Apply SF4 upgrade efforts

### PR DESCRIPTION
**Pull request by pablothedude**
When setting up the MFA GSSP project effort was made to upgrade SF4.
This effort is now added to the GSSP example project so next time a
GSSP implmentation is started the upgrade path will take less time.

### CODE REVIEW

File: homestead/after.sh [hunk 1]
```diff
@@ -7,21 +7,44 @@
...
-
+echo "\n########\n# Update PHP settings\n######\n"
+# Use our onw PHP.ini
 sudo cp /home/vagrant/code/homestead/php.ini /etc/php/7.2/mods-available/custom.ini
+
+# Use PHP 7.2 by default in our environment and enable Xdebug
 sudo phpenmod -v 7.2 custom
-COMPOSER_MEMORY_LIMIT=-1 composer install
+php72
 xon
```
Comment (MKodde) [inline homestead/after.sh]: In my project this section reads:

```
sudo phpenmod -v 7.2 custom
xon xdebug
```

I think it's important to add the xdebug after the xon statement

File: templates/base.html.twig [hunk 2]
```diff
@@ -69,38 +61,6 @@
...
-            <div><i class="fa fa-info-circle" aria-hidden="true"></i>{{ 'page.route.description.documentation_purpose'|trans }}</div>
-            <pre>{{ 'page.route.description.gssp_saml_sso_return'|trans }}</pre>
-        </div>
-    {% endif %}
-    {% if current_route == 'sp_demo' %}
-        <div class='alert'>
-            <i class="fa fa-info-circle" aria-hidden="true"></i>
-            <pre style="display: inline">{{ 'page.route.description.sp_demo'|trans }}</pre>
-        </div>
-    {% endif %}
```
Comment (MKodde) [inline templates/base.html.twig]: I removed these blocks in the Azure MFA project, but we might want to keep them in the example project..

File: templates/base.html.twig [hunk 3]
```diff
@@ -110,8 +70,6 @@
 </div>
 <div class="footer">
     <div class="footer-inner">
-        <span><a href="https://wiki.surfnet.nl/display/conextsupport/Terms+of+Service+%28NL%29"
-                 target="_blank">{{ 'page.footer.terms'|trans }}</a></span>
```
Comment (MKodde) [inline templates/base.html.twig]: No need to remove this section in my opinion

File: templates/default/index.html.twig [hunk 4]
```diff
@@ -7,8 +7,6 @@
         <p class="intro">{{ 'page.index.body.intro'|trans }}</p>
         <br>
 
-        <p>{{ 'page.index.body.sdk_discription'|trans({'%idp-entity-id%': url('gssp_saml_metadata', {})}) }}</p>
-
```
Comment (MKodde) [inline templates/default/index.html.twig]: No need to remove this section in my opinion

File: homestead/after.sh (hunk 1 above)
Comment (pablothedude) [reply homestead/after.sh]: I don't think so as `xon` is an alias for `sudo phpenmod -s cli xdebug`
`homestead/vendor/laravel/homestead/resources/aliases:14`

File: templates/base.html.twig (hunk 2 above)
Comment (pablothedude) [reply templates/base.html.twig]: :+1: will restore it

File: templates/base.html.twig (hunk 3 above)
Comment (pablothedude) [reply templates/base.html.twig]: Do you perhaps know why it was removed from MFA?

File: templates/default/index.html.twig (hunk 4 above)
Comment (pablothedude) [reply templates/default/index.html.twig]: Do you perhaps know why it was removed from MFA?

Comment (MKodde) [reply templates/default/index.html.twig]: While setting up the project, I started removing everything I thought we would not be needing. And that was one of the sdk description.

Comment (pablothedude) [reply templates/default/index.html.twig]: Thanks for the clarification, will restore it

File: templates/base.html.twig (hunk 3 above)
Comment (pablothedude) [reply templates/base.html.twig]: Thanks for the clarification, I will restore it

**Pull request merged**